### ISSUE TITLE
`no-multi-asterisks` should not trigger when there's whitespace between asterisks

### ISSUE THREAD

**Issue by edg2s** [gajus/eslint-plugin-jsdoc#803]
## Expected behavior
The sample code is allowed when using `no-multi-asterisks`.

## Actual behavior
The code is not allowed.

## ESLint Config

```js
{ "rules": {
  "no-multi-asterisks": "error"
} }
```

## ESLint sample

```js
/**
 * The method does 2 things:
 * * Thing 1
 * * Thing 2
 **/
```
```js
/**
 * This muti-line comment contains some
 * *non-standard bold* syntax
 **/
```

## Environment

- Node version: 
- ESLint version 
- `eslint-plugin-jsdoc` version: 37.0.3

**Comment by brettz9** [comment]
I think we need some config for this because this could be working as intended (to catch stray asterisks at the beginning).

**Comment by brettz9** [comment]
I would guess it may need some tweaking/refactoring, but take a look at #803

**Comment by brettz9** [comment]
Sorry had the wrong issue there... Meant #828
